### DISCUSSION TITLE
React Native Primitives 👀

### DISCUSSION THREAD

**Discussion by ahmedrowaihi** [infinitered/ignite#2699] [Ideas]
would request your genuine check this.. it would make ignite a lot closer to the community 
[rn-primitives](https://rn-primitives.vercel.app/)

**Comment by frankcalise** [comment]
What is it you're suggesting? Replacing Ignite's components with this? Or releasing Ignite's like this?

**Reply by ahmedrowaihi** [reply]
rebasing or extending Ignite's components with [rn-primitives](https://rn-primitives.vercel.app/)
what do you think boss ?

**Reply by ahmedrowaihi** [reply]
There is also this post
[post](https://x.com/mrzachnugent/status/1806413181033554240?s=46&t=OH5t3O4aiv1FkSY1_sw8xg)

**Reply by frankcalise** [reply]
You are free to give it a try in your own Ignite projects for sure, wouldn't see any reason not to!

As for it being in Ignite by default, our philosophy here is that we need to try it out on a few projects before we insert it into the boilerplate by default.

Thanks for the share though, we'll take a look at evaluating it. And let us know how it works out on your own project!

**Reply by ahmedrowaihi** [reply]
Thank you @frankcalise
